### PR TITLE
chore: added webpack

### LINKED ISSUE #676: Bundle extension
Use webpack to bundle the extension

### PR DESCRIPTION
**Pull request by JohnstonCode**
fixes #676

### CODE REVIEW

File: src/encoding.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { jschardet } from "./vscodeModules";
```
Comment (edgardmessias) [inline src/encoding.ts]: Could you try use webpack without change this?

Comment (JohnstonCode) [reply src/encoding.ts]: Yesh will try

Comment (JohnstonCode) [reply src/encoding.ts]: getting
```
WARNING in ./src/vscodeModules.ts 18:15-26
Critical dependency: the request of a dependency is an expression
 @ ./src/svn.ts
 @ ./src/extension.ts
```
I dont know enough about webpack to fix it

Comment (edgardmessias) [reply src/encoding.ts]: I will try later

**Comment by JohnstonCode** [comment]
semver 7.0 breaks webpack build.

**Pull request closed without merging**